### PR TITLE
manifest: upmerge to SOF fcd3f18b3

### PR DESCRIPTION
**Pull request by nashif**
Upmerge to SOF fcd3f18b307125fc7a671638d4612ff9ddbe7cb3.

Signed-off-by: Anas Nashif <anas.nashif@intel.com>

### CODE REVIEW

**Review by fabiobaltieri — DISMISSED** [review]

**Review by fabiobaltieri — APPROVED** [review]

**Review by fabiobaltieri — DISMISSED** [review]

**Comment by fabiobaltieri** [comment]
Hey, this is now failing `intel_adsp_cavs25` `sample.audio.sof`, looks like https://github.com/thesofproject/sof/commit/9948d439d9a9c8deff1e1fceed696fca03c7bcde enabled something that needs a submodule by default. Not sure how to fix this properly, the submodule definition does not even have a full path so it's not like we can add that module as is even if we wanted to.

**Comment by nashif** [comment]
new entries in manifest already approved: https://github.com/zephyrproject-rtos/zephyr/issues/58212

File: west.yml [hunk 1]
```diff
@@ -316,6 +316,14 @@ manifest:
...
       revision: fcd3f18b307125fc7a671638d4612ff9ddbe7cb3
       path: modules/audio/sof
+    - name: rimage
+      repo-path: rimage
+      path: modules/audio/sof/rimage
+      revision: 4fb9fe00575bc2e9f14570803d811987fb27f010
+    - name: tomlc99
+      repo-path: tomlc99
+      path: moduled/audio/sof/rimage/tomlc99
+      revision: e3a03f5ec7d8d33be705c5ce8a632d998ce9b4d1
```
Comment (fabiobaltieri) [inline west.yml:326]: can you add a MAINTAINERS.yml entry for these two?

**Review by fabiobaltieri — DISMISSED** [review]

**Comment by nashif** [comment]
looks like new entries in west need to be part of the cache!

File: west.yml [hunk 2]
```diff
@@ -314,8 +314,14 @@ manifest:
       groups:
         - debug
     - name: sof
-      revision: ee40f61b5725b6615f1abea4a78cc4d90ce144b8
+      revision: fcd3f18b307125fc7a671638d4612ff9ddbe7cb3
       path: modules/audio/sof
```
[the remark this answers is not in the document]
Comment (nashif) [reply west.yml]: sof project itself does not use that and rely on west, the submodules are there for backward compatibility only. Looking for a solution still.

**Comment by nashif** [comment]
@fabiobaltieri I reverted back to old commit + your patch and will address the manifest/module issue later.

**Comment by nashif** [comment]
@gmarull  this now includes your init.h commits as well.

**Review by fabiobaltieri — APPROVED** [review]

**Review by gmarull — APPROVED** [review]

**Comment by gmarull** [comment]
> @gmarull this now includes your init.h commits as well.

thanks

**Pull request merged**